pyproject.toml

```diff
@@ -0,0 +1,26 @@
+[tool.poetry]
+name = "IrisSoftware"
+version = "0.1.0"
+description = ""
+authors = ["Your Name <you@example.com>"]
+readme = "README.md"
+packages = [{include = "IrisSoftware"}]
+
+[tool.poetry.dependencies]
+python = ">=3.9,<3.11"
+PySide6 = "^6.3.2"
+opencv-python = "^4.6.0.66"
+PyAutoGUI = "^0.9.53"
```

Comment:
I think `PyAutoGUI` is the wrong package: https://pyautogui.readthedocs.io/en/latest/quickstart.html
It should be `pyautogui` instead

Reply:
I thought that too, but that's what it's listed as in PyPI, so it's the real deal: https://pypi.org/project/PyAutoGUI/
Trying to import PyAutoGUI does nothing, imports for pyautogui work.

Reply:
Wow, that's really weird. Frequently, malicious actors will make fake PyPI packages named as a slight misspelling of a real package, but it looks like that isn't the case here.